source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to order your query
+or aggregation operation results by string values. A collation is a set of character
+ordering conventions that apply to a specific language and locale.
+
+Collations in MongoDB
+---------------------
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Certain languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following Canadian French words: 
+
+- cote
+- coté
+- côte
+- côté
+
+When using the default binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To create a Collation object, you must specify a ``Locale`` field; all other fields are
+optional. For example, the following code snippet specifies a Collation object with
+the ``"en_US"`` locale collation:
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "en_US"}
+
+For a complete list of collation fields, visit the `Collation API documentation 
```

Comment:
Issue: I believe collation here should be monospace and capitalized as you are referring to the `Collation` type. 

```suggestion
For a complete list of ``Collation`` object fields, visit the `Collation API documentation 
```

Reply:
Made this change